kratos/geometries/geometry.h

```diff
@@ -593,7 +593,13 @@ class Geometry
     ///@{
 
     /// Id of this Geometry
-    IndexType Id() const
+    IndexType Id()
```

Comment:
you can only make this const as it does not make any change to object and returns a copy of Id.